Dockerfile

```
@@ -6,7 +6,7 @@ USER root
 RUN pip3 install git+https://github.com/enolfc/d4scienceauth.git
 
 # install oauthenticator with EGI Check-in, with refresh
-RUN pip3 install git+https://github.com/enolfc/oauthenticator@checkin
+RUN pip3 install git+https://github.com/enolfc/egiauthenticator@refresh
```

```suggestion
RUN pip3 install git+https://github.com/EGI-Foundation/egiauthenticator
```

I would directly use the one at the EGI-Foundation repo, ideally we should pin that to avoid having unexpected changes

---

mmm, nevermind this suggestion. The change is not there yet, so probably not a good idea.

---

yes I would like to switch to the EGI-foundation repo as soon as your PR on egiauthenticator is tested and merged. But to test that I would need a new image of the JupiterHub including the PR:-)